File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/SunPositionItem.cs
namespace Kendo.Mvc.Examples.Models.Chart
{
    public class SunPositionItem
    {
        public string Time { get; set; }
        public double Altitude { get; set; }
        public double Azimuth { get; set; }

        public SunPositionItem(string time, double altitude, double azimuth)
        {
            Time = time;
            Altitude = altitude;
            Azimuth = azimuth;
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/SunPositionItem.java
package com.kendoui.spring.models;

public class SunPositionItem {
    private String time;
    private double altitude;
    private double azimuth;
    
    public SunPositionItem(String time, double altitude, double azimuth) {
        setTime(time);
        setAltitude(altitude);
        setAzimuth(azimuth);
    }
    
    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
    
    public double getAltitude() {
        return altitude;
    }

    public void setAltitude(double altitude) {
        this.altitude = altitude;
    }
    
    public double getAzimuth() {
        return azimuth;
    }

    public void setAzimuth(double azimuth) {
        this.azimuth = azimuth;
    }
}
